### PR TITLE
Update SSC Softmx JITAOT test to remove unnecessary checks

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves https://github.com/eclipse/openj9-systemtest/issues/41

Signed-off-by: Mesbah Mesbah_Alam@ca.ibm.com

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Validated with: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder/142/console

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java [hunk 1]
```diff
@@ -49,40 +49,34 @@
  *  
  *  Implementation steps
  *  ~~~~~~~~~~~~~~~~~~~~
- *  1) Start 2 Jvms that run workload processes with options "-XX:SharedCacheHardLimit=20m, -Xscmxsoftmx=5m 
- *     -Xshareclasses:addtestjithints,verbose -Xscminaot30k -Xscmaxaot40k 
- *     -Xscminjitdata30k -Xscmaxjitdata40k -Xjit:count=1". 
+ *  1) Start 2 Jvms that run workload processes with options "-XX:SharedCacheHardLimit=20m, 
+ *     -Xshareclasses:addtestjithints,verbose-Xscmaxaot10k -Xscmaxjitdata10k -Xjit:count=0". 
  *
```
Comment (hangshao0) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java]: Space missing after "verbose"

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java [hunk 2]
```diff
@@ -49,40 +49,34 @@
...
  *     
- *  2) Once Jvm1 and Jvm2 finishes, check cache stats to ensure cache 
- *     is soft full. Fail if it isn't. Also make sure the AOT and JIT space is full. 
+ *  2) Once Jvm1 and Jvm2 finishes, check that the AOT and JIT space is full. 
  *     Also, record the cache stats at this point, which should give us the number of bytes
  *     written in AOT and JIT data space. 
  *     
  *  3) Start Jvm3 with with options to increase softlimit of the cache and size of the aot and jit data spaces: 
  *  	 "-Xshareclasses:adjustsoftmx=10m,adjustminaot=50k,adjustmaxaot=60k,
  *     adjustminjit=50k,adjustmaxjit=60k". Check cache stats to ensure cache's limits has been increased.
```
Comment (hangshao0) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java]: I see you removed CACHESIZE_SOFTLIMIT_INCREASED, adjustsoftmx=10m should also be removed from the comment here

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java]: Added.

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java (hunk 2 above)
Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java]: Good Catch! Updated comment.

**Comment by Mesbah-Alam** [comment]
@smlambert - Please review and merge this PR when you have a moment.

**Pull request merged**